pyClient/zeth/mixer_client.py

```diff
@@ -657,9 +673,11 @@ def compute_h_sig(
 
 def trap_r_randomness() -> str:
     """
-    Compute randomness "r" as 48 random bytes
+    Compute randomness "r" as 32 random bytes
     """
-    return bytes(Random.get_random_bytes(32)).hex()
+    assert (constants.TRAPR_LENGTH & 7) == 0
+    trapr_len_bytes = constants.TRAPR_LENGTH >> 3
+    return bytes(Random.get_random_bytes(trapr_len_bytes)).hex()
```

Comment:
Here I used constants when fixing some issues during rebase.  However, there seems to be several other places where constants can be used.  #203